analyzers/rspec/vbnet/S6608.html

```diff
@@ -0,0 +1,272 @@
+<h2>Why is this an issue?</h2>
+<p>Indexes in C# provide direct access to an element at a specific position within an array or collection. When compared to <code>Enumerable</code>
+methods, indexing can be more efficient for certain scenarios, such as iterating over a large collection, due to avoiding the overhead of checking the
+underlying collection type before accessing it.</p>
+<p>This applies to types that implement one of these interfaces:</p>
+<ul>
+  <li> <a href="https://learn.microsoft.com/en-us/dotnet/api/system.collections.ilist">IList</a> </li>
```

Comment:
As suggested in [this comment](https://github.com/SonarSource/sonar-dotnet/issues/7125#issuecomment-1535227192) I think we should add `IReadOnlyList<T>` both on rspec and implementation

Reply:
Added it here, also check the [latest commit](https://github.com/SonarSource/rspec/pull/1787/commits/13d797937b70eec9c1d86a682baf525e3839932d) on the RSPEC PR :)